src/super_gradients/recipes/cityscapes_ddrnet.yaml

```diff
@@ -55,7 +55,17 @@ architecture: ddrnet_23
 
 training_hyperparams:
   max_epochs: 500
-  initial_lr: 0.0075   # batch size 24
+  initial_lr: # batch size 24
+    default: 0.075
```

Comment:
Second thought. Do we want to use `default` or `_default_` (Similar to `_target_`) ?  
Not leaning towards either option tbh

Reply:
I prefer "default" as _default_ might imply this argument is hydra related.